inference-engine/tests/ie_test_utils/functional_test_utils/layer_test_utils.hpp

```diff
@@ -34,6 +34,13 @@ typedef std::tuple<
         std::string                  // Target Device
 > basicParams;
 
+typedef std::tuple<
+    InferenceEngine::Precision,         // Network Precision
+    InferenceEngine::SizeVector,        // Input Shape
+    std::string,                        // Target Device
+    std::map<std::string, std::string>  //Configuration
+> basicParamsWithConfig;
```

Comment:
@iefode @mikhail-treskin What do you think about this? Should we contribute additional configs to `LayerTestsUtils` ?